apps/website/src/components/UnstableApiCard.astro

```diff
@@ -0,0 +1,12 @@
+---
+import WarningIcon from '~/icons/WarningIcon.astro';
+import DocsNoteCard from './DocsNoteCard.astro';
+---
+
+<DocsNoteCard {...Astro.props}>
+  <WarningIcon aria-hidden='true' slot='icon' />
+  <p slot='content'>
+    <strong>Unstable</strong>: This component is marked unstable to help us gather feedback and
```

Comment:
I just realized that we are not actually asking for feedback.

Maybe we can create a "Feedback" category on GitHub Discussions, and link it here?

Reply:
We could also repurpose the existing ["Documentation" cateogory](https://github.com/iTwin/iTwinUI/discussions/categories/documentation).

If we do this, will need to also update the `FeedbackLink.astro` component to point to the new category.